core/editor/editor.php

```diff
@@ -867,6 +868,7 @@ public function set_edit_mode( $edit_mode ) {
 	 */
 	public function __construct() {
 		Plugin::$instance->data_manager_v2->register_controller( new Data\Globals\Controller() );
+		Plugin::$instance->data_manager_v2->register_controller( new Connect_Controller() );
```

Comment:
Move to the connect module

Reply:
Wordpress REST endpoints are registered globally, and are requested globally. Therefore we register them in the editor class.